src/hidimstat/permutation_test.py

```diff
@@ -1,25 +1,18 @@
 import numpy as np
 from joblib import Parallel, delayed
 from sklearn.base import clone
-from sklearn.model_selection import GridSearchCV
-from sklearn.pipeline import Pipeline
-from sklearn.svm import LinearSVR
-from sklearn.utils import _safe_indexing
-
-from hidimstat.stat_tools import pval_from_two_sided_pval_and_sign
-
-
-def permutation_test_cv(
-    X,
-    y,
-    n_permutations=1000,
-    C=None,
-    Cs=np.logspace(-7, 1, 9),
-    seed=0,
-    n_jobs=1,
-    verbose=1,
+
+from hidimstat.stat_tools import pval_from_two_sided_pval_and_sign, step_down_max_t
+
+
+def permutation_test(
+    X, y, estimator, n_permutations=1000, seed=0, n_jobs=-1, verbose=0
```

Comment:
```suggestion
    X, y, estimator, n_permutations=1000, seed=0, n_jobs=-1, verbose=0
```
I just have one more question: why do you set the default `n_jobs=-1`? 
I have a preference for =1 since `n_jobs>1` makes debugging tedious and setting it to -1 may lead to the overuse of shared computing resources. WDYT?

Reply:
This function is quite slow; I more in tendency to use most of the computational available for running it. Moreover, most of the time, there will be more than 20 permutations; thus the resources won't be overused.

However, I don't have a strong opinion on it. Scikit-learn used by default `None` for allowing a global configuration due to the back-end of joblib. I modified it like them to be more compatible with Scikitlearn.